Cargo.toml

```diff
@@ -12,7 +12,7 @@ leptos_meta = { version = "0.6" }
 leptos_router = { version = "0.6" }
 leptos_i18n = "0"
 cfg-if = "1"
-lemmy-client = { version = "0.19.4-rc.7", features = ["leptos"] }
+lemmy-client = { version = "1.0.0", features = ["leptos"] }
```

Comment:
[See this relevant issue if you're confused about the version change.](https://github.com/LemmyNet/lemmy-client-rs/issues/11)